### DISCUSSION TITLE
Support AddItem(s) and RemoveItem(s)

### DISCUSSION THREAD

**Discussion by dirtyes** [asticode/go-astisub#119] [Ideas]
Helper methods like:

- Append
- Prepend
- Insert
- Remove

**Comment by lindowx** [comment]
I had the same request, but it looks the project was not maintanced any more.
So, I was plan to make fork it and make some optimizes.

**Comment by asticode** [comment]
> I had the same request, but it looks the project was not maintanced any more.

This project is still actively maintained. I don't have time to answer all issues and I'm also relying on PRs for the biggest changes.
